src/main/steps/respondent/international-factors/summary/content.ts

```diff
@@ -111,12 +111,19 @@ const urls = {
   request: INTERNATIONAL_FACTORS_REQUEST,
 };
 
+const fieldType = {
+  start: 'String',
+  parents: 'String',
+  jurisdiction: 'String',
+  request: 'String',
+};
+
 const cy: typeof en = (content: CommonContent) => {
   const userCase = content.userCase!;
   return {
     ...cyContent,
     language: content.language,
-    sections: [summaryList(cyContent, userCase, urls, 'respondentAdditionalInformation')],
+    sections: [summaryList(cyContent, userCase, urls, 'International elements', fieldType, content.language)],
```

Comment:
Can we pass variable 'enContent.title2' as 3rd parameter which we are already defining above.

Reply:
There is 's' in the end for International elements

Reply:
code is updated with  'cyContent.title2' as 3rd parameter